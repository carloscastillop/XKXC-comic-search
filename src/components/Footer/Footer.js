import React from 'react';
import style from './Footer.module.scss';

const Footer = () => {
    return (
        <footer
            className={style.footer}
        >
            <div
                className={`${style['container']} ${style['text-muted']}`}
            >
                <p>XKXC comic search</p>
            </div>
        </footer>
    );
};

export default Footer
