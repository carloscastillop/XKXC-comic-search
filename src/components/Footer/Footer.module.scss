@import '../../styles/global.module.scss';

.footer {
  margin-top: $spacer * 2;
  padding: ($spacer * 2) $spacer 100px $spacer;
  text-align: center;
  background-color: #171717;

  p {
    font-size: 80%;
    margin: 0;
    color: rgba(127, 127, 127, 0.3);
  }
}