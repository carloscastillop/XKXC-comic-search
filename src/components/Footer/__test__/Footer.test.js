import React from 'react';
import {render} from '@testing-library/react';
import Footer from '../Footer';

test('Check name app footer', () => {
    const {getByText} = render(<Footer/>);
    const linkElement = getByText(/XKXC comic search/i);
    expect(linkElement).toBeInTheDocument();
});
