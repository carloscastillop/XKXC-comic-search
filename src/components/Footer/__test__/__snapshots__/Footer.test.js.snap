// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshop 1`] = `
<footer
  className="footer"
>
  <div
    className="container text-muted"
  >
    <p>
      XKXC comic search
    </p>
  </div>
</footer>
`;
